src/manifest.json

```diff
@@ -41,10 +42,13 @@
     "templates/custom_styles.css",
     "data/modules_i.json",
     "data/modules_ics.json",
-    "data/modules_wi.json"
+    "data/modules_wi.json",
+    "_locales/*"
   ],
+  "default_locale": "de",
   "browser_specific_settings": {
     "gecko": {
+      "id": "{64b1a494-88df-447e-b2aa-45d86fcc3c16}",
```

Comment:
What's that?

Reply:
The ID of the extension, the same you find in de update.json
It's needed for sync storage. I think i accidentally merged it from the branch: [extensionSync  ](https://github.com/eddex/hslu-simple-mep-results/tree/enhancement/extensionSync)